config/samples/oran.openshift.io_v1alpha1_clusterrequest.yaml

```diff
@@ -22,17 +22,17 @@ spec:
         - NTP.server1
         - 1.1.1.1
       baseDomain: example.com
-      clusterLabels:
-        common: "true"
-        group-du-sno: test
-        sites: site-sno-du-1
       clusterName: site-sno-du-1
       extraAnnotations:
         AgentClusterInstall:
           extra-annotation-key: extra-annotation-value
       extraLabels:
         AgentClusterInstall:
           extra-label-key: extra-label-value
+        ManagedCluster:
```

Comment:
I would update these to labels that we are actually using, to have an example that's closer to reality:
```
    cluster-version: v4.16
    clustertemplate-a-policy: v1
```